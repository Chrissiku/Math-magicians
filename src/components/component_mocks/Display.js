/* eslint-disable react/prop-types */
/* eslint-disable react/prefer-stateless-function */
/* eslint-disable react/destructuring-assignment */
import React from 'react';

class Display extends React.Component {
  render() {
    // eslint-disable-next-line react/prop-types
    return (
      <div className="result">
        <div className="number">{this.props.value}</div>
      </div>
    );
  }
}

export default Display;
