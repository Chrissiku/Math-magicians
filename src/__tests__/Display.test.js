/* eslint-disable react/jsx-props-no-spreading */
import renderer from 'react-test-renderer';
import Display from '../components/component_mocks/Display';

const props = {
  value: {
    total: 0,
    next: '',
    operation: '',
  },
};

it('Test Display function ', () => {
  const TREE = renderer.create(<Display {...props} />).toJSON;
  expect(TREE).toMatchSnapshot();
});
