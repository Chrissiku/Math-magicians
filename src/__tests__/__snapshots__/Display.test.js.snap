// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Display function  1`] = `[Function]`;
